ironic/resource_ironic_node_v1.go

```diff
@@ -628,3 +664,14 @@ func (workflow *provisionStateWorkflow) changeProvisionState(target nodes.Target
 
 	return false, nodes.ChangeProvisionState(workflow.client, workflow.d.Id(), *opts).ExtractErr()
 }
+
+func changePowerState(client *gophercloud.ServiceClient, id string, target nodes.TargetPowerState) error {
+	if err := nodes.ChangePowerState(client, id, nodes.PowerStateOpts{
+		Target:  target,
+		Timeout: 30,
```

Comment:
Are you sure you want to provide the timeout by default?

Reply:
This is still a valid concern. Timeout requires a newer microversion and the server defaults may be good enough.